### PR TITLE
[RM Ch7.3]: Adding ETSI NFV SEC WG

### PR DESCRIPTION
**Pull request by CsatariGergely**
- Adding a link to the ETSI NFV SEC WG specifications
- Some corrections on how non ordered lists are done in .md

fixes: #596

### CODE REVIEW

File: doc/ref_model/chapters/chapter07.md [hunk 1]
```diff
@@ -82,43 +82,30 @@ Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor i
...
-• ISO (the International Organization for Standardization) and IEC (the International Electrotechnical Commission) - www.iso.org.  The following ISO standards are of particular interest for NFVI
-
-    o ISO/IEC 27002:2013 - ISO/IEC 27001 is the international Standard for best-practice information security management systems (ISMSs).
-
-    o ISO/IEC 27032 - ISO/IEC 27032is the international Standard focusing explicitly on cybersecurity.
-
-    o ISO/IEC 27035 - ISO/IEC 27035 is the international Standard for incident management. Incident management
-
-    o ISO/IEC 27031 - ISO/IEC 27031 is the international Standard for ICT readiness for business continuity.
+Standards organizations with recommendations and best practices, and certifications that need to be taken into consideration include the following examples. However this is by no means an exhaustive list, just some of the more important standards in current use. [inde](http://index.hu)
```
Comment (deebhatia) [inline doc/ref_model/chapters/chapter07.md]: \[inde](http://index.hu) what is this?

Comment (ASawwaf) [reply doc/ref_model/chapters/chapter07.md]: @deebhatia , :)  we can remove it, the clause itself not clear we can remove the clause

Comment (CsatariGergely) [reply doc/ref_model/chapters/chapter07.md]: Uh-oh, sorry I was just testing the syntax of linking ;)

Comment (CsatariGergely) [reply doc/ref_model/chapters/chapter07.md]: Corrected now.  Thanks for pointing it out.

Comment (deebhatia) [reply doc/ref_model/chapters/chapter07.md]: > Uh-oh, sorry I was just testing the syntax of linking ;)

No worries. 😄

**Comment by rabiabdel** [comment]
@kedmison please review/approve

**Pull request merged**